synchronicity/combined_types.py

```diff
@@ -56,10 +58,14 @@ def __get__(self, instance, owner=None):
 
 
 class AsyncAndBlockingContextManager(typing_extensions.Protocol[CTX]):
-    def __enter__(self) -> CTX: ...
+    def __enter__(self) -> CTX:
+        ...
```

Comment:
I think we should be able to preserve the previous formatting for these stubs with ruff (based on https://github.com/astral-sh/ruff/pull/6592); do you know why that's not happening here?